README.md

```diff
@@ -1,19 +1,77 @@
 # Submit a GAR Public Site UI
 
-## Docker Usage
-The following depends on a .env file in the root of this project with the required environment variables. This will start a local test instance of keycloak for local testing.
+## Requirements
+- docker
+- docker-compose
+- node
+- npm
+- your email added to GOV Notify Dev whitelist by repository administrator
+
+## Development
+
+The following depends on a `.env.dev` file in the root of this project with
+the required environment variables. To obtain it please contact repository administrator.
+
+This will start a local instances of:
+- Database (used for both Frontend and Backend)
+- API (you need a local copy of `api` docker image loaded, 
+       please contact repository administrators to obtain it)
+- ClamAV engine
+- ClamAV REST API 
+- Frontend
+
+Please note - due to security configuration, the following lines need to be 
+commented out for local development, but not committed back to the repository:
+```
+./src/server.js (lines 95-98):
```

Comment:
The fix would require:
- reverse proxy running
- an external certificate to be installed inside reverse proxy
It would involve some work and associated cost. There is no time and budget for that in Sprint 12 unfortunately